lib/vagrant-openshift/helper/command_helper.rb

```diff
@@ -141,10 +141,27 @@ def sync_bash_command_on_dockerfile(repo_name, dockerfile_build_path, build_cmd)
         sync_bash_command(repo_name, build_cmd, file_path, branch_name, sync_path, commit_id_path)
       end
 
+      def check_and_fix_repo(repo_path)
```

Comment:
I'm not saying that, I'm saying that the repo needs to be known-good or the repo directory needs to be deleted by [this test](https://github.com/openshift/vagrant-openshift/pull/485/files#diff-a1bd9ead90d4dfc0eff8f7783369031aR165) because the content of the command will vary based on that result.